libs/langchain/tests/unit_tests/document_loaders/parsers/test_public_api.py

```diff
@@ -14,4 +14,5 @@ def test_parsers_public_api_correct() -> None:
         "PyMuPDFParser",
         "PyPDFium2Parser",
         "PDFPlumberParser",
+        "ZeroxPDFParser",
```

Comment:
could you remove this

Reply:
No problem. Can you explain?
ZeroxPDFParser seems to be missing from the list.